### DISCUSSION TITLE
Displaying two images

### DISCUSSION THREAD

**Discussion by gkim132** [jspsych/jsPsych#802] [Q&A] [answered]
Hi,

I'm trying to display two images and let subjects pick one image. I thought "stimulus: [ 'a.png' , 'b.png'] " would work, but it didn't work.

Any help will be greatly appreciated!

Thank you!

**Comment by vijaymarupudi** [comment]
I think what the author wrote in this discussion post will be helpful!

https://github.com/jspsych/jsPsych/discussions/757

**Reply by gkim132** [reply]
Thank you!

**Comment by jodeleeuw** [comment, marked as answer]
Hi @gkim132,

You can use HTML to put two images on the screen:

```js
var trial = {
	type: 'html-keyboard-response',
	stimulus: '<img src="a.png"></img>'+
		'<img src="b.png"></img>'
}
```

**Reply by gkim132** [reply]
Is there another way using type: 'image-keyboard-response' ?

**Reply by jodeleeuw** [reply]
No, the `image-keyboard-response` plugin is for a single image. The `html-keyboard-response` is intended for cases where you want more customization than is possible with the `image-keyboard-response`.
